react/features/recording/components/LiveStream/StartLiveStreamDialog.native.js

```diff
@@ -70,6 +70,46 @@ class StartLiveStreamDialog extends AbstractStartLiveStreamDialog {
         this._onStreamKeyChange(streamKey);
     }
 
+    _onStreamKeyPick: string => void
+
+    /**
+     * Callback to be invoked when the user selects a stream from the picker.
+     *
+     * @private
+     * @param {string} streamKey - The key of the selected stream.
+     * @returns {void}
+     */
+    _onStreamKeyPick(streamKey) {
+        this.setState({
+            streamKey
+        });
+    }
+
+    _onUserChanged: Object => void
+
+    /**
+     * A callback to be invoked when an authenticated user changes, so
+     * then we can get (or clear) the YouTube stream key.
+     *
+     * @private
+     * @param {Object} response - The retreived signin response.
+     * @returns {void}
+     */
+    _onUserChanged(response) {
+        if (response && response.user) {
+            googleApi.getYouTubeLiveStreams(response).then(broadcasts => {
+                this.setState({
+                    broadcasts
+                });
+            });
```

Comment:
Is it worthwhile to have error handling right now if getting the live streams fails?